ui/src/app/auth/auth.guard.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable }           from '@angular/core';
+import { Router, CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot }  from '@angular/router';
+import { AuthService }          from './auth.service';
```

Comment:
I think it's too hard to try and line up imports. I think we could organize them like we do in the Go code, library components at the top, blank line and custom components below that, and then not worry about space aligning the `from` after the brackets.